modules/gitops-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-6_{context}"]
+= Release notes for {gitops-title} 1.8.6
+
+{gitops-title} 1.8.6 is now available on {OCP} 4.10, 4.11, 4.12 and 4.13.
```

Comment:
```suggestion
{gitops-title} 1.8.6 is now available on {OCP} 4.11, 4.12 and 4.13.
```
@harrietgrace A question regarding the OCP support version for 1.8.6:

Now that 4.10 is out of support, can you confirm that I can remove 4.10 from here and make 4.11 the lowest-supported OCP version for 1.8.6? Thank you!

Reply:
Though 4.10 is out of support, we decided to standardise on all patch versions having the same support bracket as their minor version. I'd like to retain the 4.10 listing here on the 1.8.x z stream for reference.

Reply:
Sure @harrietgrace, will retain it. Thanks for the confirmation. @masauso-lungu Please note.

Reply:
Noted.